Tests/Packet++Test/Tests/CotpTests.cpp

```diff
@@ -0,0 +1,39 @@
+#include "../TestDefinition.h"
+#include "../Utils/TestUtils.h"
+#include "CotpLayer.h"
+#include "Packet.h"
+#include "SystemUtils.h"
+
+using namespace std;
+
+PTF_TEST_CASE(CotpLayerTest)
+{
+	timeval time;
+	gettimeofday(&time, nullptr);
+
+	READ_FILE_AND_CREATE_PACKET(1, "PacketExamples/cotp.dat");
+
+	pcpp::Packet CotpLayerTest(&rawPacket1);
```

Comment:
variable names should begin with lower case, also I'd rename it to: `cotpPacket`